src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/DESCryptoServiceProvider.Unix.cs

```diff
@@ -84,5 +84,7 @@ public override PaddingMode Padding
             get { return _impl.Padding; }
             set { _impl.Padding = value; }
         }
+
+        protected override void SetKeyCore(ReadOnlySpan<byte> key) => _impl.SetKey(key);
```

Comment:
If this was anything other than DES and RC2 I would suggest making our `SetKeyCore` `protected internal` so the `SetKeyCore` could just call `SetKeyCore` again, thus avoiding going trough the public API twice.

But, it's DES and RC2. So it's probably fine.